### PR TITLE
fix: v1 release date

### PR DESCRIPTION
**Pull request by Justintime50**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## v1.0.0 FUTURE
+## v1.0.0 (2023-11-20)
```
Comment (jchen293) [inline CHANGELOG.md:3]:
```suggestion
## v1.0.0 (2023-11-21)
```

Comment (nwithan8) [reply CHANGELOG.md:3]: Version was released yesterday, 11-20.

**Review by nwithan8 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jchen293) [reply CHANGELOG.md:3]: Ooo, didn't see that

**Review by jchen293 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Justintime50) [reply CHANGELOG.md:3]: This was released yesterday

**Pull request merged**